Increase consistency of named objects across case studies

Schiano-NOAA opened the pull request:
<!---
Thanks for opening a PR. This commented text will **NOT** appear in the final PR. Toggle between Write and Preview to see what your PR will look like without the comments.
-->

# What is the feature?
* #68 

# How have you implemented the solution?
* Moved files to appropriate folders
* change names of qmd cases when needed to match naming conventioned
* added in section to the case studies template to tell user how to name their file
* reorganized case study files into own folder
* changed path of .qmd's in webpage yaml so it does not break

# Does the PR impact any other area of the project, maybe another repo?
* yes - changed pathing for webpage

kellijohnson-NOAA commented:
Thank you for providing the guidance to authors in the template on how to name their file. Rather than including it as text though, I think that we should include it as a comment. I found this [guidance](https://github.com/quarto-dev/quarto-cli/discussions/3330#discussioncomment-4146233) on how to include a comment. If you could change it to a comment, that would be great. Second, #66 provided guidance to remove files, one of which was removed in this PR, but not to move files. Thus, in order for me to merge this PR the file location for each of the case studies needs to be reverted. Please let me know if these modifications are something that you have time to do. No worries if the answer is no. Thanks for playing around in the repo and learning about the FIMS case studies 😀!

Schiano-NOAA commented:
@kellijohnson-NOAA I converted the guidance for file naming into a comment as suggested and moved files back to their previous location. Sorry that we moved them around, I definitely missed the part where it said not to move them.

Thanks for reviewing and let me know if any additional changes need to be made! Happy to do it

kellijohnson-NOAA commented:
Thanks Workflows team for this PR.
